blocks/bricks/recurrent.py

```diff
@@ -962,18 +948,18 @@ def apply(self, *args, **kwargs):
         low_memory : bool
             Use the slow, but also memory efficient, implementation of
             this code.
-
-        See docstring of the class for arguments appearing in
-        self.apply.sequences, self.apply.states, self.apply.contexts
-        All arguments values are of type :class:`~tensor.TensorVariable`.
-
-        In addition the `iterate`, `reverse`, `return_initial_states` or
-        any other argument defined in `recurrent_apply` wrapper.
+        sequences, states, contexts : :class:`~tensor.TensorVariable`
+            See docstring of the class for arguments appearing in the
+            lists
+            self.apply.sequences, self.apply.states, self.apply.contexts
+        iterate, reverse, return_initial_states : bool
```

Comment:
Please use `See Also` section for such purpose, e.g.

```
See Also
------------
:func:`~blocks.brick.recurrent.recurrent` : for all other parameters
```

Also a `Notes` section could mention that `reverse` is currently not supported.